recipes/lora_finetune_single_device.py

```diff
@@ -554,6 +554,8 @@ def _loss_step(self, batch: Dict[str, torch.Tensor]) -> torch.Tensor:
         labels = labels[..., 1:].contiguous()
         logits = logits.transpose(1, 2)
         # Compute loss
+        torch._dynamo.mark_dynamic(logits, 2)
+        torch._dynamo.mark_dynamic(labels, 1)
```

Comment:
does this matter if we are not compiling loss_step anymore?

Reply:
yes this influence how `self._loss_fn` is compiled (currently for per-layer compile, we also compile the loss_fn)